theme/less/_header.less

```diff
@@ -0,0 +1,69 @@
+.header-top
+{
+  padding: 25px;
+}
+
+header label select
+{
+  width: 163px;
+  color: #3A7D11;
+  font-size: 24px;
+  border: 0;
+  opacity: 0.7;
+  appearance: none;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  background: url(/images/language-arrow.png) no-repeat right;
+}
+
+header a
+{
+  text-decoration: none;
+}
+
+.header-top a
+{
+  max-width: 190px;
+}
+
+header p
+{
+  display: inline;
+  opacity: 0.7;
+  color: #000000;
+  padding: 0 10px;
+}
+
+header img
+{
+ vertical-align: middle;
+ width: 32px;
+ height: auto;
+}
+
+#hamburger
+{
+  background-color: transparent;
+  cursor: pointer;
+  border: 0;
+  background-image: url(/images/hamburger.svg);
+  width: 32px;
+  height: auto;
+}
+
+@media (max-width: @tablet-width)
+{
+  header
+  {
+    border-top: 1px solid #E9E9E9;
+    background: #FFFFFF;
+    box-shadow: 0px 8px 15px rgba(0, 0, 0, 0.06);
+  }
+
+  header label select
+  {
+    font-size: 13px;
+    width: 100px;
+    padding: 10px;
+  }
```

Comment:
This style is also not obviously belongs to the header, I wonder if it's a better idea not to separate header and navigation, to avoid such issues?